examples/data_preprocessing/mimic_dataset/prepare_mimic_cxr.ipynb

```diff
@@ -0,0 +1,1520 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#merge records, labels and split\n",
+    "record_report_split_list = pd.merge(record_report_list, split_list, how = 'left', on = ['dicom_id', 'study_id','subject_id'])\n",
+    "record_report_split_label_list = pd.merge(record_report_split_list, labels_chexpert.iloc[:,[0,1,16]], how = 'left', on = ['study_id','subject_id'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The train/validation/test split is taken from mimic-cxr-jpg, which is intended to provide a reference dataset split for studies using this data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "train-val-test split proportions: split\n",
+      "test        0.013680\n",
+      "train       0.978388\n",
+      "validate    0.007931\n",
+      "dtype: float64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"train-val-test split proportions:\" ,record_report_split_label_list.groupby('split').size()/len(record_report_split_label_list))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "train-val-test split proportions: split\n",
+      "test        0.013680\n",
+      "train       0.978388\n",
+      "validate    0.007931\n",
+      "dtype: float64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"train-val-test split proportions:\" ,record_report_split_label_list.groupby('split').size()/len(record_report_split_label_list))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are 14 classes, where about 42% of the studies are assigned to the class 'No finding'. We should keep in mind that we are dealing with an unbalanced dataset.\n",
+    "\n",
+    "Now the labels get replaced by their id. This gives us the final \"input_list\". Since we will need this file multiple times and the runtime is relatively long, it is cached."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_list_full = record_report_split_label_list"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['input_list.lib']"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "#save the whole file\n",
+    "dump(input_list_full, \"input_list.lib\")"
```

Comment:
the names if the output files whould be defined in the very beginning as well

Reply:
Where would be a good place for that? Before or after the library imports?